### ISSUE TITLE
jq colors inappropriate for white backgrounds

### ISSUE THREAD

**Issue by lconly** [jqlang/jq#2924]
**Describe the bug**
Version 1.7 of jq uses white for non json keys/values which do not appear on white backgrounds.

**To Reproduce**
Use a terminal with a white background and view any jq output (in color).

**Expected behavior**
There should be standard support for white backgrounds or at least some instructions on how to set the JQ_COLORS
environment variable to the pre 1.7 colors.

**Environment (please complete the following information):**
macOs 14.0
jq version 1.7
iTerm2 terminal with light color scheme

**Additional context**
This was a change in the default colors for 1.7.

**Comment by wader** [comment]
Hi, might be dup of https://github.com/jqlang/jq/issues/2903

**Comment by lconly** [comment]
Yes, same basic issue just more profound; with the solarized background the white text is hard to read; with a white background it's not visible.

**Comment by wader** [comment]
Could you try this which should be the new default colors that was merged to master:
```
echo '{"a": 1, "b": "b", "c": [1,2,3,4,5]}' | JQ_COLORS="0;90:0;39:0;39:0;39:0;32:1;39:1;39:1;34" jq
```

**Comment by wader** [comment]
Ok just remembered that i'm using iTerm2 also :D

<img width="643" alt="Screenshot 2023-10-06 at 17 35 42" src="https://github.com/jqlang/jq/assets/185566/6014cc30-367b-4f99-b7f9-96593f92e7b4">

**Comment by emanuele6** [comment]
This can be closed, I guess.

@lconly To clarify, this is exactly the same problem as #2903; `jq 1.7` hardcodes the default colour of those elements to white (`37`) instead of using the foreground colour of the terminal's colour theme (`39`); #2904 fixed this by changing the default.

In the mean time, you can add `JQ_COLORS="1;30:0;39:0;39:0;39:0;32:1;39:1;39"` to your environment, which is jq's default colour scheme, but with `37` replaced by `39`.

**Comment by Cimbali** [comment]
Sorry − is 1.8 going to be released soon? If not, could this fix be backported/released in a 1.7.x patch? This bug is a major annoyance to me and, I suspect, to all of the intersection of white-background terminal users and jq enthusiasts.

**Comment by wader** [comment]
@Cimbali Yes i agree a 1.7.1 release would be good i think. I've started to update NEWS.md but it seems the other maintainers are busy so might take some time.
